src/main/java/com/uci/utils/azure/AzureBlobService.java

```diff
@@ -182,16 +178,23 @@ public String uploadFile(String urlStr, String mimeType, String name) {
 	
 				/* File input stream to copy from */
 				URL url = new URL(urlStr);
-				InputStream in = url.openStream();
-	
+				byte[] inputBytes = url.openStream().readAllBytes();
+
+				/* Discard if file size is greater than MAX_SIZE_FOR_MEDIA */
+				Double maxSizeForMedia = Double.valueOf(System.getenv("MAX_SIZE_FOR_MEDIA"));
```

Comment:
Try sending it via the method attributes, as it may differ for different files. It can be 2 MB for image, 3 MB for audio etc.
and if there is no max file size given we should not add this restriction here.